service/matching/task_queue_partition_manager_test.go

```diff
@@ -113,6 +113,33 @@ func (s *PartitionManagerTestSuite) TestAddTaskNoRules_AssignedTask() {
 	s.validatePollTask(bld, true)
 }
 
+func (s *PartitionManagerTestSuite) TestDescribeTaskQueuePartition_UnloadedVersionedQueues() {
```

Comment:
would this test fail if you had not made changes to the code?

Reply:
yes, the test fails if I these changes don't get merged in.